lib/Main.js

```diff
@@ -63,6 +61,28 @@ function Main(config, registration) {
         registration,
         roomStore,
         userStore,
+        roomUpgradeOpts: {
+            migrateEntry: (entry, newId) => {
```

Comment:
Where is this called? I can't see any references elsewhere

Reply:
https://github.com/matrix-org/matrix-appservice-bridge/blob/master/lib/components/room-upgrade-handler.js#L120